systems/WikibaseClient/06-Runtime_View.md

```diff
@@ -2,6 +2,8 @@
 
 ## UpdateRepoHookHandler
 
+see the [Wikipedia editor's use case](01-Introduction_and_Goals.md#wikipedia-editor): _Connect pages on different Wikipedias_
```

Comment:
This one isn't quite right. I don't think we have a runtime diagram for connecting a page, and we don't have a use case that relates to `UpdateRepoHookHandler`. 

Connecting a page is a prerequisite for the use case involving `UpdateRepoHookHandler`, which is what happens when an already connected page is moved or deleted.

Reply:
Do you think we should add a use case "keep interwiki links up to date, when pages are moved or deleted"?
Or rather add to the existing one: "Connect pages on different Wikipedias, showing interwiki links on the connected pages and keeping them up to date when pages are moved or deleted"?

Reply:
This is somewhat similar to embedding Entity data in client wiki content, for which we have two stakeholder expectations:

> * Use data from a Wikibase Repository in Wikibase Client wikitext generated content.
> * Have data from a Wikibase Repository used in Wikibase Client pages update automatically when changes happen on the Wikibase Repository.

So I think it would be consistent to add it as a separate bullet.